src/sections/hero.html

```diff
@@ -1,4 +1,10 @@
-<div>
-    <h1 class="text-7xl">Hero</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+
+<div class="relative flex h-screen justify-center items-center">
+    <img src="/src/img/Ira_kelia-shape.svg" alt="Shape" class="h-auto
+    w-[100%]">
+    <p class="absolute left-16 text-white text-2xl font-semibold">We help you set
+        focus and gain
+        impact. By adding
+        to the conversation,
+        not to the noise.</p>
```

Comment:
I would suggest writing it like this for better code readability!
```suggestion
   <p class="absolute left-16 text-white text-2xl font-semibold">
        We help you set focus and gain impact. By adding to the conversation, not to the noise.
    </p>
```